### PR TITLE
Code Refactored

### PR DESCRIPTION
**Pull request by rojandahal**
-Input and Button component separated for signup page

### CODE REVIEW

File: src/pages/Authentication/Signup/Signup.jsx [hunk 1]
```diff
@@ -136,37 +129,33 @@ export default function Signup() {
...
-                className="bg-black text-white"
-              />
-            </FormGroup>
+            <InputField
+              req={true}
+              forLabel="What should we call you?"
+              placeholder="Enter a profile name."
+              handleOnChange={(e) => setUsername(e.target.value)}
+              type="text"
+              className="bg-black text-white"
```
Comment (aayussraut) [inline src/pages/Authentication/Signup/Signup.jsx]: - yo repeat bhairaxa ..yeslai ne utai iInputField ko component ma rakha

File: src/pages/Authentication/Signup/Signup.jsx [hunk 2]
```diff
@@ -136,37 +129,33 @@ export default function Signup() {
...
-                type="text"
-                onChange={(e) => setUsername(e.target.value)}
-                className="bg-black text-white"
-              />
-            </FormGroup>
+            <InputField
+              req={true}
+              forLabel="What should we call you?"
+              placeholder="Enter a profile name."
+              handleOnChange={(e) => setUsername(e.target.value)}
```
Comment (aayussraut) [inline src/pages/Authentication/Signup/Signup.jsx]: yesko lagi ne euta handleChange bhanne function bhanayera pass gara ez hunxa paxe kunai field add ganru paryo bhane

**Review by aayussraut — APPROVED** [review]
- tyo milau ani merge gardeu

File: src/pages/Authentication/Signup/Signup.jsx (hunk 1 above)
Comment (rojandahal) [reply src/pages/Authentication/Signup/Signup.jsx]: Okay

File: src/pages/Authentication/Signup/Signup.jsx (hunk 2 above)
Comment (rojandahal) [reply src/pages/Authentication/Signup/Signup.jsx]: Changes Made

**Pull request merged**